### ISSUE TITLE
Accessibility. Elements must only use allowed ARIA attributes: ARIA attribute is not allowed: aria-selected="false"

### ISSUE THREAD

**Issue by rokalm** [ng-select/ng-select#1983]
**Describe the bug**
Group elements that have attribute role="group" cannot have aria-selected attribute according to ARIA standards.
https://accessibilityinsights.io/info-examples/web/aria-allowed-attr/

**More info** 
https://www.levelaccess.com/how-not-to-misuse-aria-states-properties-and-roles/
Only use aria-selected with these roles: option, tab, menuitemradio, treeitem, gridcell, row, rowheader and columnheader.

**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):
1. Go to https://ng-select.github.io/ng-select#/grouping
2. Open dropdown
3. Inspect source code of group element
